publishedbranches/docs-mongocli.yaml

```
@@ -27,8 +27,8 @@ version:
     - '1.2'
     - '1.1'
     - '1.0'
-  stable: '1.8'
-  upcoming: '1.9'
+  stable: 'v1.9'
+  upcoming: '1.10'
```

I know `upcoming` isn't a used field here, but maybe this should be `v1.10` for consistency?
```suggestion
  upcoming: 'v1.10'
```

---

I think that's smart! Thanks for catching it